src/screens/SettingsModal.js

```diff
@@ -98,6 +91,12 @@ const SettingsPages = {
     getTitle: () => lang.t('settings.network'),
     key: 'NetworkSection',
   },
+  notifications: {
+    component:
+      IS_DEV || isTestFlight ? DevNotificationsSection : UserDevSection,
```

Comment:
I suspect that this line is what's breaking e2e because you're adding the UserDevSection twice.

component should be null if falsey since it won't be rendered at all.

Reply:
Also,  why aren't we moving this inside Developer Settings menu?

That will still show on testflight....  See https://github.com/rainbow-me/rainbow/pull/3329/files#diff-37b761430eed85cee253642e8d1980af44f75a75e5b6dbb2d88a1598a5bd7fddR80

@welps would that work for you?

Reply:
> Also, why aren't we moving this inside Developer Settings menu?
> That will still show on testflight.... See https://github.com/rainbow-me/rainbow/pull/3329/files#diff-37b761430eed85cee253642e8d1980af44f75a75e5b6dbb2d88a1598a5bd7fddR80
> @welps would that work for you?

yes

Reply:
alright let me move it there